### DISCUSSION TITLE
Can you please fix the "Select Category" dropdown to not require JavaScript?

### DISCUSSION THREAD

**Discussion by BugHunter1000** [community/community#3131] [Discussions] [answered]
First of all thanks for the new feature and for supporting this great platform for sharing ideas.

I am all for JS being used when it is called for.  A web player, crypto libraries for clientside encryption, online gaming, and so on.  Please do not misunderstand this as a "JS is the devil" post, but I also like to keep my life simple and elegant where I can.

Toward this, I avoid JS where it is not called for, which greatly improves my computer's performance and stability.  I have noticed a noble and significant effort on the part of the Github devs to support a JS-free experience.  And if I may rant for a moment:

This is a fucking text forum.

Can you please fix the bug in Select Category where I have to turn JS on to do that one thing?  I realize your UI devs are going to screech and moan that OH I CANT STYLE DROPDOWNS WOE IS ME OH BROWSER COMPATIBILITY OH MY LIFE IS TERRIBLE but just give a noscript fallback.  You're literally forcing us to use the category feature, I 100% cannot post a discussion without it, and if you're going to be pissy about it and cop attitude and strongarm users that we are utterly not allowed to post without using this feature, don't strangle us into using a needless JS hack.  It really isn't that hard, and it's the right solution.

/rant

Thank you for all you do, and happy new year.

**Comment by evi-liu** [comment, marked as answer]
Hi there and happy new year to you, too. Unfortunately, we currently use JavaScript across our GitHub platform. This is not unique to Discussions and we don't have any plans to remove JS.
